modules/javafx.controls/src/main/java/javafx/scene/control/Toggle.java

```diff
@@ -104,4 +104,10 @@ public interface Toggle {
      * by application developers
      */
     ObservableMap<Object, Object> getProperties();
+
+    /**
+     * Tests if Toggle has properties.
+     * @return true if node has properties.
+     */
```

Comment:
```suggestion
    /**
     * Tests if this toggle has properties.
     *
     * @return {@code true} if this toggle has properties
     * @since 22
     */
```